job_controller/job_controller.go

```diff
@@ -60,6 +73,42 @@ type ControllerInterface interface {
 
 	// Returns the Job from API server
 	GetJobFromAPIClient(namespace, name string) (metav1.Object, error)
+
+	// Returns the default container in pod. e.g. tensorflow
+	GetDefaultContainerName() string
+
+	// Returns the default container port name. e.g. tfjob-port
+	GetDefaultContainerPortName() string
+
+	// DeleteJob deletes the job
+	DeleteJob(job interface{}) error
```

Comment:
similarl to DeleteJob, is there a CreateJob ?

Reply:
CreateJob is done by the user, when creating a CRD (i.e. TFJob). operators will not create its own CRDs